graylog2-server/src/test/java/org/graylog2/shared/bindings/providers/OkHttpClientProviderTest.java

```diff
@@ -92,7 +92,7 @@ public void testSuccessfulProxyConnectionWithoutAuthentication() throws IOExcept
         assertThat(server.getRequestCount()).isEqualTo(1);
         final RecordedRequest recordedRequest = server.takeRequest();
         assertThat(recordedRequest.getMethod()).isEqualTo("GET");
-        assertThat(recordedRequest.getPath()).isEqualTo("http://www.example.com/");
+        assertThat(recordedRequest.getPath()).isEqualTo("/");
```

Comment:
@dennisoelkers What is the reason why these URLs/paths had to be changed?

Reply:
This is related to the introduction of https://github.com/square/okhttp/pull/4235 in `3.12.0`, which is fixing path parsing for IPv6. Actually it makes sense, because `http://www.example.com` is not part of the request path.

This is only related to the tests, the actual OkHttp client code has not changed in this regard.